tools/mtouch/xlf/Errors.ru.xlf

```diff
@@ -2223,7 +2231,14 @@
 		</source>
         <target state="new">The type '{0}' (used as {1} {2}) is not available in {3} {4} (it was introduced in {3} {5}){6} Please build with a newer {3} SDK (usually done by using the most recent version of Xcode).
 		</target>
-        <note>
+        <note>This is a message for usage of a type that is not available targeting older platform versions. The following are literal names and should not be translated: SDK, Xcode
+{0} - The full C# type name.
+{1} - One of MT4162_BaseType, MT4162_Parameter, MT4162_ReturnType, or MT4162_PropertyType
```

Comment:
This might not be possible to translate properly. There could be languages where these inserted parts consist of multiple words that should go in different parts of the sentence, which would make it impossible because there's a single {1}.

I think a better approach would be to have 4 complete error messages, instead of composing one at runtime.

So:

* MT4162_BaseType: The type '{0}' (used as _a base type_ of {2}) is not available in {3} {4} (it was introduced in {3} {5}){6} Please build with a newer {3} SDK (usually done by using the most recent version of Xcode).
* MT4162_Parameter: The type '{0}' (used as _a parameter in_ {2}) is not available in {3} {4} (it was introduced in {3} {5}){6} Please build with a newer {3} SDK (usually done by using the most recent version of Xcode).
...